tests/Webapi/Webapi__Dom/Webapi__Dom__Window__test.re

```diff
@@ -58,7 +58,7 @@ let _ = Window.getSelection(window);
 let _ = Window.matchMedia("max-height: 400", window);
 let _ = Window.moveBy(10, (-10), window);
 let _ = Window.moveTo(120, 300, window);
-let _ = Window.open_(~url="http://...", ~name="my window", ~features="menubar=yes", window);
+let _ = Window.open_(~url="http://...", ~name="my window", ~features=?Some("menubar=yes"), window);
```

Comment:
Then you don't need to make any change here, the argument can be passed like before but also left out.